src/cow-react/modules/limitOrders/updaters/QuoteUpdater/hooks/useQuoteRequestParams.ts

```diff
@@ -7,11 +7,14 @@ import { useMemo } from 'react'
 import { useTypedValue } from '@cow/modules/limitOrders/hooks/useTypedValue'
 import { getAddress } from '@cow/modules/limitOrders/utils/getAddress'
 import useENSAddress from 'hooks/useENSAddress'
-import { useLimitOrdersDeadline } from '@cow/modules/limitOrders/hooks/useLimitOrdersDeadline'
+import ms from 'ms.macro'
+
+// It's request to get price, so we don't need precise value for validTo
+const PRICE_QUOTE_VALID_TO_TIME = ms`30m`
 
 export function useQuoteRequestParams(): FeeQuoteParams | null {
   const { inputCurrency, outputCurrency, recipient, orderKind } = useLimitOrdersTradeState()
-  const deadlineTimestamp = useLimitOrdersDeadline()
+  const deadlineTimestamp = Math.round((Date.now() + PRICE_QUOTE_VALID_TO_TIME) / 1000)
```

Comment:
I understood from the thread with Nick that when quoting for limit orders we shouldn't pass a validTo at all?

Reply:
Thanks! Removed it